index.js

```diff
@@ -76,9 +76,14 @@ exports.install = function(){
   var self = this;
   massive.connect({db : DB}, function(err,db){
     var sql = self.build();
+    if(!db || !db.run) {
+      console.log("Unable to connect to database '" + DB + "'.");
+      console.log("Check that it exists and/or adjust connection settings in index.js");
+      return;
```

Comment:
I'm not sure if the `return` statement here is enough or if we also need to prevent `this.install()` below.